release-prep.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+docker run -it --rm -e CHANGELOG_GITHUB_TOKEN -v $(pwd):/usr/local/src/your-app \
```

Comment:
`your-app` seems a little bit suspect here, is it supposed to be replaced with maybe `beaker-puppet`?

Reply:
I think that's okay--from a quick look at other repos that use this automation, it seems like that path is local to the container and more or less a temporary staging area for the changelog after it's been generated.

Tefoji ran a release prep action a few weeks ago: https://github.com/puppetlabs/tefoji/actions/runs/7732718060/job/21083254959

Which shows this in the logs:

```
 Generated log placed in /usr/local/src/your-app/CHANGELOG.md
```

And created this PR: https://github.com/puppetlabs/tefoji/pull/107